### DISCUSSION TITLE
L2 error for a transient simulation

### DISCUSSION THREAD

**Discussion by pharshaad** [idaholab/moose#18568] [Q&A General] [answered]
Hi all,

I'm following example 14 for calculating the L2 norm of the error for a transient wave propagation simulation. I'm using a fine mesh solution as my exact solution, however, I'm a bit confused with the multiple .xda and _mesh.xda files that are being generated for the fine mesh solution. I'm not sure which one to feed into the SolutionUserObject of the coarse mesh simulation. Is it going to be the last file (0200.xda and 0200_mesh.xda for a 200 timestep run)?

Thanks,
Ari

**Comment by GiudGiud** [comment, marked as answer]
You can specify the highest number (so the latest timestep) or you can replace the number with .LATEST in your input file and it ll pick the latest. 

Guillaume
